source/reference/footnotes.txt

```diff
@@ -0,0 +1,44 @@
+=========
+Footnotes
+=========
+
+Use footnotes to add detail that not all readers will need.
```

Comment:
Suggestion: I think its worth mentioning that footnotes render at the very bottom of the page later. And in your example below, you can add a few lines of regular text to really showcase how it works.

Reply:
Just making sure I understand:
1. I think footnotes render wherever you put the Footnotes rubric, which is why I mentioned putting it at the bottom of the page.
2. You mean adding some sample text in the example before the footnote rubric?

Reply:
Yes! both correct! I just figured some dummy text between where you create the foodnote reference and the actual footnote section really showcases how to use them.

Reply:
Is lorem ipsum okay? Or is there a preferred way to create dummy text?

Reply:
Yea that'd be fine. Or to make it even simpler, you can make it something like: 

<your footnote references>

This describes something cool.
This goes into more detail. 
Now you've learned this cool thing!

<the footnotes rendered>

Reply:
love it